### DISCUSSION TITLE
Anyone can help me with explainint this "state.counter.value" syntax

### DISCUSSION THREAD

**Discussion by saurabhkushwaha438** [reduxjs/react-redux#2149] [General]
so basically what is role of counter and if change its name to like mycounter it show error why ,despite i dont declare it anywhere

**Comment by markerikson** [comment]
Please actually read through our tutorials:

- https://redux.js.org/tutorials/essentials/part-2-app-structure
- https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers
- https://redux-toolkit.js.org/usage/immer-reducers
